isthmus/build.gradle.kts

```diff
@@ -1,10 +1,13 @@
 plugins {
+  `maven-publish`
```

Comment:
Is backticks really the right thing? (here and elsewhere)?

Reply:
I was wondering the same. That's how it's defined in the docs though it doesn't work otherwise.